ucore/sanoid.spec

```diff
@@ -0,0 +1,141 @@
+%global version 2.2.0
+%global git_tag v%{version}
+
+# Enable with systemctl "enable sanoid.timer"
+%global _with_systemd 1
+
+Name:		   sanoid
+Version:	   %{version}
+Release:	   1%{?dist}
+BuildArch:	   noarch
+Summary:	   A policy-driven snapshot management tool for ZFS file systems
+Group:		   Applications/System
+License:	   GPLv3
+URL:		   https://github.com/jimsalterjrs/sanoid
+Source0:	   https://github.com/jimsalterjrs/%{name}/archive/%{git_tag}/%{name}-%{version}.tar.gz
+
+Requires:	   perl-interpreter, mbuffer, lzop, pv, perl-Config-IniFiles, perl-Capture-Tiny, perl-Sys-Hostname
+%if 0%{?_with_systemd}
+Requires:      systemd >= 212
+
+BuildRequires: systemd
+%endif
+
+%description
+Sanoid is a policy-driven snapshot management
+tool for ZFS file systems. You can use Sanoid
+to create, automatically thin, and monitor snapshots
+and pool health from a single eminently
+human-readable TOML configuration file.
+
+%prep
+%setup -q
+
+%build
+echo "Nothing to build"
+
+%install
+%{__install} -D -m 0644 sanoid.defaults.conf %{buildroot}/etc/sanoid/sanoid.defaults.conf
+%{__install} -d %{buildroot}%{_sbindir}
+%{__install} -m 0755 sanoid syncoid findoid sleepymutex %{buildroot}%{_sbindir}
+
+%if 0%{?_with_systemd}
+%{__install} -d %{buildroot}%{_unitdir}
+%endif
+
+%if 0%{?fedora}
+%{__install} -D -m 0644 sanoid.conf %{buildroot}%{_docdir}/%{name}/examples/sanoid.conf
+%endif
+%if 0%{?rhel}
+%{__install} -D -m 0644 sanoid.conf %{buildroot}%{_docdir}/%{name}-%{version}/examples/sanoid.conf
+%endif
+
+%if 0%{?_with_systemd}
+cat > %{buildroot}%{_unitdir}/%{name}.service <<EOF
+[Unit]
+Description=Snapshot ZFS Pool
+Requires=zfs.target
+After=zfs.target
```

Comment:
I notice that this service doesn't match CentOS recommendations found in sanoid docs: https://github.com/jimsalterjrs/sanoid/blob/master/INSTALL.md#centos

Maybe we should consider adding the missing `sanoid-prune.service` from docs?

Reply:
I agree with this and if you do as well, I can submit another commit with these additions as well.

Reply:
I agree, we should add `sanoid-prune.service`.

Please and thank you!

Reply:
This has been added.